plugin/src/main/java/com/redhat/jenkins/plugins/ci/messaging/ActiveMqMessagingWorker.java

```diff
@@ -432,12 +432,17 @@ public SendResult sendMessage(Run<?, ?> build, TaskListener listener, ProviderDa
                 message.setText(PluginUtils.getSubstitutedValue(pd.getMessageContent(), envVars2));
 
 
-                publisher.send(message);
-                log.info("Sent " + pd.getMessageType().toString() + " message for job '" + build.getParent().getName() + "' to topic '" + ltopic + "':\n"
+                String messageType = "unknown type";
+                if (pd.getMessageType() != null) {
```

Comment:
Do we ever want to send a message when the type is null?

Reply:
if sent via a pipeline step, it is currently possible to omit the message type. I was seeing NPEs from a log statement where type was NULL.